pygments/lexers/ldap.py

```diff
@@ -0,0 +1,84 @@
+"""
+    pygments.lexers.ldap
+    ~~~~~~~~~~~~~~~~~~~~
+
+    Pygments lexers for LDAP.
+
+    :copyright: Copyright 2006-2023 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+import re
+
+
+from pygments.lexer import RegexLexer, bygroups
+from pygments.token import Operator, Comment, Keyword, Name, String, \
+    Number, Punctuation, Whitespace, Escape
+
+__all__ = ['LdifLexer']
+
+
+class LdifLexer(RegexLexer):
+
+    """
+    Lexer for LDIF
+
+    .. versionadded:: 2.17
+    """
+
+    name = 'LDIF'
+    aliases = ['ldif']
+    filenames = [ "*.ldif" ]
+    url = "https://datatracker.ietf.org/doc/html/rfc2849"
+
+    tokens = {
+       'root': [
+            (r'\s*\n', Whitespace),
+            (r'-\n', Punctuation),
+            (r'(#.*)(\n)', bygroups(Comment.Single, Whitespace)),
+            (r'(version)(:)([ \t]*)([^\n]*)([ \t]*\n)', bygroups(Keyword, Punctuation, Whitespace, Number.Integer, Whitespace)),
+            (r'(control)(:)([ \t]*)([\.0-9]+)([ \t]+)((?:true|false)?)([ \t]*\n)', bygroups(Keyword, Punctuation, Whitespace, Name.Constant, Whitespace, Keyword, Whitespace)),
```

Comment:
That's actually not an integer but on OID (something like `1.2.840.113556.1.4.805`).

I have been using `Name` for that because in the following rules, the first parameter is either a name (`cn`) or an OID (`2.5.4.3`) and I have types this as `Name.Attribute` in both cases (which may be questionnable):

~~~
            (r'([a-zA-Z]*|[0-9][0-9\.]*[0-9])(;)', bygroups(Name.Attribute, Punctuation), "property"),
            (r'([a-zA-Z]*|[0-9][0-9\.]*[0-9])(:<)', bygroups(Name.Attribute, Punctuation), "url"),
            (r'([a-zA-Z]*|[0-9][0-9\.]*[0-9])(::?)', bygroups(Name.Attribute, Punctuation), "value"),
~~~

Reply:
Using `Name.Other` for now.